tests/Oryx.RuntimeImage.Tests/Oryx.RuntimeImage.Tests.csproj

```diff
@@ -24,6 +24,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\src\BuildScriptGenerator\BuildScriptGenerator.csproj" />
+    <ProjectReference Include="..\Oryx.BuildImage.Tests\Oryx.BuildImage.Tests.csproj" />
```

Comment:
Why is this being referenced? Kind of weird to reference BuildImage tests project in runtimeimages test

Reply:
it's not needed, I will clean it .. thanks